Programm started at: 2025-09-08 02:02:36

Running in version 1.2[2025-09-08 23:32:12]
Git repositories synchronized
	

	NoneType: None


[2025-09-08 23:32:12]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-09-09 02:01:58

Running in version 1.2

[2025-09-09 23:32:32]
Git repositories synchronized
	

	NoneType: None


[2025-09-09 23:32:32]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-09-10 02:02:58

Running in version 1.2

[2025-09-10 23:32:33]
Git repositories synchronized
	

	NoneType: None


[2025-09-10 23:32:33]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-09-11 02:02:35

Running in version 1.2

[2025-09-11 23:32:09]
Git repositories synchronized
	

	NoneType: None


[2025-09-11 23:32:09]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-09-12 02:01:59

Running in version 1.2

[2025-09-12 23:32:34]
Git repositories synchronized
	

	NoneType: None


[2025-09-12 23:32:34]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-09-13 02:02:58

Running in version 1.2

[2025-09-13 03:08:59]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


